docs/installation/cli-config.rst

```diff
@@ -0,0 +1,7 @@
+.. _install_cli-config:
+
+KSQL CLI Configuration
```

Comment:
Consistency nit: Server section says "Configuring KSQL Server", so this section here should be "Configuring KSQL CLI"